### ISSUE TITLE
NVDA screen reader not reading values correctly and reading as "Blank"

### ISSUE THREAD

**Issue by dk-massive** [highcharts/highcharts#20783]
It appears that there are issues with the NVDA reader having the ability to use the various charts. The most common item being that the first value is reading as "Blank".

Below are the various charts tested and their issues. Each on is provides a link to a codepen for testing.

[Area Chart ](https://codepen.io/dk-massive/full/MWRwZNr)- (using NVDA) only first chart value can be tabbed to and is read as “blank.” Remaining chart values cannot be tabbed to. No values read upon mouse hover. All data points are read when chart is navigated to from “landmarks” tree in elements list, but data points aren’t read in item order as displayed on chart.

[Bar Chart](https://codepen.io/dk-massive/full/JjVdVZB) - (using NVDA) only first chart value can be tabbed to and is read as “blank.” Remaining chart values cannot be tabbed to. No values read upon mouse hover. All data points are read when chart is navigated to from “landmarks” tree in elements list, but data points aren’t read in item order as displayed on chart.

[Bubble Chart](https://codepen.io/dk-massive/full/OJGVGBP) - (using NVDA) only first chart value can be tabbed to and is read as “blank.” Remaining chart values cannot be tabbed to. No values read upon mouse hover. Data points are not read when navigated to from “landmarks” tree in elements list.

[Column Chart](https://codepen.io/dk-massive/full/xxeGemJ) - (using NVDA) Data points in chart cannot be tabbed or arrowed through. Values are read upon mouse hover. Data points are read when chart is navigated to from “landmarks” tree in elements list, but some values not displayed on the chart are read.

[Gauge Chart](https://codepen.io/dk-massive/full/wvZaZOG) - (using NVDA) Only first chart value can be tabbed to--value is not read but label is. Remaining values cannot be tabbed to. Only “20" value is read upon mouse hover. Data points are read when chart is navigated to from “landmarks” tree in elements list. Some additional values (1010, 3030, 2020) are read that do not exist in back end- seems to be repeating values without reading the labels of items.

[Line Chart](https://codepen.io/dk-massive/full/bGJdyVg) - (using NVDA) Only first chart value can be tabbed to--value is read as “blank”. Remaining values cannot be tabbed to. No data values are read upon mouse hover. No data point values are read when chart is navigated to from “landmarks” tree in elements list.

[Pie Chart](https://codepen.io/dk-massive/full/gOypJpa) - (using NVDA) Only first chart value can be tabbed to--value is read as “blank”. Remaining values cannot be tabbed to. No data values are read upon mouse hover. No data point values are read when chart is navigated to from “landmarks” tree in elements list.

[Scatter Chart](https://codepen.io/dk-massive/full/rNbVgxK) - (using NVDA) Only first chart value can be tabbed to--value is read as “blank”. Remaining values cannot be tabbed to. No data values are read upon mouse hover. No individual data point values are read when chart is navigated to from “landmarks” tree in elements list.

[Spline Chart ](https://codepen.io/dk-massive/full/MWRwdyV)- (using NVDA) Only first chart value (item 2 yellow diamond) can be tabbed to--value is read as “blank”. When tabbing back, value of data point will be read. Remaining values cannot be tabbed to. No data values are read upon mouse hover. No individual data point values are read when chart is navigated to from “landmarks” tree in elements list.
